react/ForceCenter.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+
+const ForceCenter = ({ children }) => (
+  <div
+    className="absolute left-0 right-0 flex justify-center z-1"
+    style={{ pointerEvents: 'none' }}
```

Comment:
Why is that? Just out of curiosity

Reply:
because it creates a transparent div with 100% of the width, which might cover elements behind it and render the user unable to click through